src/libs/OptionsListUtils.js

```diff
@@ -952,7 +954,7 @@ function getOptions(
     }
 
     let currentUserOption = _.find(allPersonalDetailsOptions, (personalDetailsOption) => personalDetailsOption.login === currentUserLogin);
-    if (searchValue && !isSearchStringMatch(searchValue, currentUserOption.searchText)) {
+    if (searchValue && currentUserOption && !isSearchStringMatch(searchValue, currentUserOption.searchText)) {
```

Comment:
NAB: Why this change?

Reply:
good question! that's for a fix for [this one](https://expensify.slack.com/archives/C05DWUDHVK7/p1693506335746079?thread_ts=1693492764.933499&cid=C05DWUDHVK7)

I was getting an error when `currentUserOption` is undefined and the code tries to read `currentUserOption.searchText`.

[Although I don't know why this suddenly stopped working](https://expensify.slack.com/archives/C05DWUDHVK7/p1693515077643159?thread_ts=1693492764.933499&cid=C05DWUDHVK7)

Reply:
Let's figure this out in a follow up